loops_lab.js

```diff
@@ -0,0 +1,108 @@
+//Write a while loop and a for loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+
+//while loop
+let num = 10
+while (num <= 10 && num >= 1){
+  console.log(num);
+  num --
+}
+//for loop
+for (let num2 = 10; num2 <= 10 && num2 >= 1; num2 --){
```

Comment:
For for loops, it is not recommended to put two conditions (like (num2 <= 10 && num2 >= 1)). It is better to just put one condition and make it like this (for (let num2 = 10; num2 >= 1; num2 --)).